packages/cxl-ui/src/components/cxl-accordion-card.js

```diff
@@ -1,12 +1,31 @@
 import { customElement } from 'lit-element';
 import '@conversionxl/cxl-lumo-styles';
 import { AccordionPanelElement } from '@vaadin/vaadin-accordion/src/vaadin-accordion-panel';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+
+import cxlAccordionCardCSS from '../styles/global/cxl-accordion-card-css';
+
+registerStyles(
+  'cxl-accordion-card',
+  css`
+    [part='summary'] {
+      padding: 0;
+      font-size: var(--lumo-font-size-xs);
+      font-weight: 500;
+    }
+  `
+);
```

Comment:
Do not obfuscate styles by distributing them around in unexpected places.

Does any other thing in this repo use `registerStyles()` with an inline stylesheet?

Reply:
moved to external `scss` https://github.com/conversionxl/aybolit/pull/172/commits/6d6910d090802c9530a93a9f415d78d99058168c